pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -570,6 +754,52 @@ func newNginx(instance *v1alpha1.RpaasInstance, plan *v1alpha1.RpaasPlan, config
 			Lifecycle:       instance.Spec.Lifecycle,
 		},
 	}
+
+	if plan.Spec.Config.CacheHeaterEnabled {
+		initCmd := defaultCacheHeaterCmdPVCToPod
+		if len(plan.Spec.Config.CacheHeaterSync.CmdPVCToPod) > 0 {
+			initCmd = plan.Spec.Config.CacheHeaterSync.CmdPVCToPod
+		}
+
+		n.Spec.PodTemplate.Volumes = append(n.Spec.PodTemplate.Volumes, corev1.Volume{
+			Name: "cache-heater-volume",
+			VolumeSource: corev1.VolumeSource{
+				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+					ClaimName: instance.Name + cacheHeaterVolumeSuffix,
+				},
+			},
+		})
+
+		cacheHeaterVolume := corev1.VolumeMount{
+			Name:      "cache-heater-volume",
+			MountPath: "/var/cache/cache-heater",
```

Comment:
This string `"/var/cache/cache-heater"` is used in multiple places, I think a constant for it would be nice.